week4_HW/mytodo-app/src/stores/DifferentValues.js

```diff
@@ -0,0 +1,27 @@
+
+import {observable, action} from 'mobx'
+
+class DifferentValues {
```

Comment:
Why did you create a separate store for this? If it works, that's OK, I am just wondering your reasoning behind this.